deployment/deployment.json

```diff
@@ -7,35 +7,35 @@
           "steegecs": "steegecs/abracadabra-money-arbitrum",
           "messari": "messari/abracadabra-money-arbitrum",
           "dmelotik": "dmelotik/abracadabra-arbitrum",
-          "deploy-on-merge": true
+          "deploy-on-merge": false
```

Comment:
Changing this file was intended?

Reply:
I just wanted to ensure we wouldn't deploy subgraphs and lose progress since there are changes in individual folders

Reply:
Ah right, that makes sense